ipn/ipnlocal/local.go

```diff
@@ -128,6 +128,14 @@ func RegisterNewSSHServer(fn newSSHServerFunc) {
 	newSSHServer = fn
 }
 
+// watchSession represents a WatchNotifications
+// channel along with its sessionID in order
+// notifiers to close targeted buses.
```

Comment:
nit: could improve the wording here. 

```suggestion
// watchSession represents a WatchNotifications channel
// and sessionID as required to close targeted buses.
```